src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,64 @@
+package command;
+
+import exception.DeadlineException;
+import exception.DukeParseException;
+import storage.Storage;
+import task.Deadline;
+import task.Task;
+import task.TaskList;
+import ui.Ui;
+
+import java.text.ParseException;
+
+/**
+ * Command for creating a Deadline task.
+ */
+public class DeadlineCommand extends Command {
+
+    protected String[] input;
+
+    public DeadlineCommand(String[] input) {
+        this.input = input;
+    }
+
+    /**
+     * Execute creating Deadline task.
+     * Output what is needed.
+     * @param tasks the TaskList.
+     * @param ui the User Interface which responsible for every output printing.
+     * @param storage user's hard disk storage.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeParseException, DeadlineException {
+        if (input.length <= 1) {
+            throw new DeadlineException();
+        } else if (input.length <=4 ) {
+            throw new DeadlineException("OOPS!!! Did you type something wrong?\n"

```

Comment:
very good that you have different error messages for the different types of errors!